slobrok/src/vespa/slobrok/server/exchange_manager.cpp

```diff
@@ -53,6 +50,7 @@ ExchangeManager::removePartner(const std::string & name)
     auto oldremote = std::move(_partners[name]);
     LOG_ASSERT(oldremote);
     _partners.erase(name);
+    oldremote->shutdown();
```

Comment:
I was confused about not having an unsubscribe after this one, but realized that we do not have a consensus map just yet. Consider adding a comment.